lib/util/common.ts

```diff
@@ -59,3 +61,23 @@ export function detectPlatform(
 
   return null;
 }
+
+export function parseJsonWithFallback(content: string): any {
+  let parsedJson: any;
+
+  try {
+    parsedJson = JSON.parse(content);
+  } catch (err) {
+    try {
+      parsedJson = JSON5.parse(content);
+      logger.warn(
+        'JSON5.parse was used to parse the JSON data. Please check your json file'
+      );
+    } catch (err) {
+      logger.warn('Invalid JSON format');
```

Comment:
```suggestion
```

I'm not sure we need this? We don't log currently if it fails do we?

Reply:
We do in some cases and don't in others 
Eg. 
 `platform.getJsonFile` functions return null in case of an error with no log message
 `parsePresets` log `invalid preset JSON`, and 
 `getParsedContent` logs   `Could not parse config file`